resources/views/support/components/image-gallery.blade.php

```diff
@@ -1,62 +1,79 @@
+                }
+                return $image_asset;
+            }
+        @endphp
+        <div class="image-gallery__wrap">
+            <section class="collection-carousel splide p-0">
+                <div class="splide__arrows splide__arrows--ltr">
+                    <button class="splide__arrow splide__arrow--prev carousel-control-prev" type="button" aria-label="Previous slide" aria-controls="splide01-track">
+                        <svg height="48" viewBox="0 0 48 48" width="64" xmlns="http://www.w3.org/2000/svg">
+                            <path fill="black" d="M0 0h48v48h-48z"></path>
+                            <path fill="white" d="M14.83 30.83l9.17-9.17 9.17 9.17 2.83-2.83-12-12-12 12z"></path>
+                        </svg>
+                    </button>
+                    <button class="splide__arrow splide__arrow--next carousel-control-next" type="button" aria-label="Next slide" aria-controls="splide01-track">
+                        <svg height="48" viewBox="0 0 48 48" width="64" xmlns="http://www.w3.org/2000/svg">
+                            <path fill="black" d="M0 0h48v48h-48z"></path>
+                            <path fill="white" d="M14.83 30.83l9.17-9.17 9.17 9.17 2.83-2.83-12-12-12 12z"></path>
+                        </svg>
+                    </button>
                 </div>
-                <button class="carousel-control-prev" type="button" data-bs-target="#image-gallery" data-bs-slide="prev">
-                    <svg height="48" viewBox="0 0 48 48" width="64" xmlns="http://www.w3.org/2000/svg">
-                        <path fill="black" d="M0 0h48v48h-48z"></path>
-                        <path fill="white" d="M14.83 30.83l9.17-9.17 9.17 9.17 2.83-2.83-12-12-12 12z"></path>
-                    </svg>
-                    <span class="visually-hidden">Previous</span>
-                </button>
-                <button class="carousel-control-next" type="button" data-bs-target="#image-gallery" data-bs-slide="next">
-                    <svg height="48" viewBox="0 0 48 48" width="64" xmlns="http://www.w3.org/2000/svg">
-                        <path fill="black" d="M0 0h48v48h-48z"></path>
-                        <path fill="white" d="M14.83 30.83l9.17-9.17 9.17 9.17 2.83-2.83-12-12-12 12z"></path>
-                    </svg>
-                    <span class="visually-hidden">Next</span>
-                </button>
-            </div>
+                <div class="splide__track">
+                    <div class="splide__list">
+                        @foreach($image_gallery['gallery_images'] as $image)
+                            <li class="splide__slide">
+                                @php
+                                    $current_image = getImageData($image_source, $image['image_id']);
+                                @endphp
+                                @if(!empty($image['image_caption'])) <figure> @endif
+                                    @if(!empty($current_image))
```

Comment:
It would be better to set this to:

```
@if(!empty($current_image['data']['full_url']))
```

Otherwise we could end up with unusual errors where `$current_image` is not empty, but `$current_image['data']['full_url']` is empty (I think this would result in a critical error)